src/universal/modules/teamDashboard/components/Team/Team.js

```diff
@@ -26,8 +28,14 @@ const Team = (props) => {
     team
   } = props;
   if (!team) return <LoadingView />;
-
-  const {teamId, teamName, isPaid, meetingId} = team;
+  const {contentFilter, teamId, teamName, isPaid, meetingId} = team;
+  const updateFilter = (e) => {
+    const nextValue = e.target.value;
+    commitLocalUpdate(atmosphere, (store) => {
```

Comment:
@dan-f this replaces a redux reducer. no boilerplate necessary

Reply:
On the other hand:

- This trades away the abstraction which the action/reducer boilerplate provides
  - `<Team />` now is coupled to details the relay store
- Blurring the concept of GraphQL as an interface for pulling data from a service. Eyeing the query below, it's not clear that `contentFilter` is a special local field.

Reply:
would you rather it be a redux reducer? or have state owned by the Action component & passed down via context or many chains of props?

Reply:
Meh, I'm cool with the current approach. Just wanted to air the tradeoffs!